src/util-memcmp.c

```diff
@@ -33,163 +33,140 @@
 #ifdef UNITTESTS
 #include "util-debug.h"
 
-static int MemcmpTest01 (void)
+static int MemcmpTest01(void)
```

Comment:
Please avoid making unnecessary formatting changes as a part of this PR. This should only contain FAIL/PASS API changes.